web-frontend-basic/style.css

```diff
@@ -0,0 +1,611 @@
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: var(--font-primary);
+  background-color: var(--white-color);
+  color: var(--black-color);
+  font-weight: 400;
+  line-height: 1.6;
+  font-size: 1.6rem;
+}
+
+img {
+  max-width: 100%;
+}
+
+button {
+  font-family: var(--font-primary);
+  border: 0;
+  font-size: 2rem;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+::placeholder {
+  font-family: var(--font-primary);
+  font-weight: 300;
+}
+
+/* Form */
+
+.form {
+  max-width: 100%;
+}
+
+.form-input {
+  font-weight: 700;
+  display: inline-block;
+  padding: 1.4rem 2.6rem;
+  border-radius: 10rem;
+  border: 0;
+}
+
+/* Price */
+
+.price {
+  font-size: 2rem;
+  font-weight: 400;
+  display: flex;
+  justify-content: space-between;
+}
+
+.price.new-price {
+  color: var(--red-color);
+}
+
+.price.old-price {
+  color: var(--gray-100-color);
+}
+
+/* Section */
+
+section {
+  padding: 4rem 0;
+}
+
+.section-title {
+  font-size: 3.2rem;
+  font-weight: 300;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.section-actions {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+/* List group */
+
+.list-group-item {
+  margin: 1.2rem 0;
+}
+
+.list-group-title {
+  font-weight: 600;
+}
+
+.list-group-link {
+  color: var(--white-color);
+  font-weight: 300;
+}
+
+/* Button */
+
+.btn,
+.btn:link,
+.btn:visited {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 1.3rem;
+  font-weight: 700;
+  display: inline-block;
+  padding: 1.4rem 2.6rem;
+  border-radius: 10rem;
+  color: var(--black-color);
+}
+
+.btn-outline:link,
+.btn-outline:visited {
+  border: 1px solid var(--gray-200-color);
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-white {
+  background-color: var(--white-color);
+}
+
+/* Badge */
+
+.badge {
+  position: absolute;
+  top: 2rem;
+  left: 2rem;
+  background-color: var(--red-color);
+  padding: 0.7rem 1.4rem;
+  border-radius: 100rem;
+}
+
+.badge span {
+  color: var(--white-color);
+  font-weight: 600;
+}
+
+/* State */
+
+.text-center {
+  text-align: center;
+}
+
+/* Header */
+
+header {
+  position: absolute;
+  width: 100%;
+  top: 4rem;
+  z-index: 100;
+}
+
+.navbar-logo-img {
+  width: 14rem;
+}
+
+.nav-list,
+.nav-actions-list {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list {
+  gap: 8rem;
+  justify-content: center;
+}
+
+.nav-link:link,
+.nav-link:visited {
+  color: var(--primary-color);
+  font-weight: 600;
+  font-size: 1.8rem;
+}
+
+.nav-actions-list {
+  gap: 2rem;
+  justify-content: flex-end;
+}
+
+.btn-icon-img {
+  width: 2rem;
+}
+
+/* Section banner */
+.section-banner {
+  background-image: url("./assets/hero-img.jpg"),
+    linear-gradient(150deg, rgba(0, 0, 0, 0.8), rgba(0, 0, 0, 0));
+  background-blend-mode: overlay;
+  background-size: cover;
+  background-position: center;
+  position: relative;
+  margin-bottom: 5rem;
+  box-shadow: 0.5rem 0.5rem 2rem rgba(0, 0, 0, 0.2);
+  z-index: -100;
```

Comment:
Rank A: Không được set thuộc tính này vì  button trong banner không click được. Nếu để fix cái background của benefit thì hoàn toàn không cần thiết.